MobileCenter/MobileCenter/Internals/MSServiceInternal.h

```diff
@@ -11,13 +11,6 @@
 @protocol MSServiceInternal <MSService, MSServiceCommon>
```

Comment:
`+@property(nonatomic, copy, readonly) NSString *groupID;` Should be there too as required so that any feature must implement it.